apps/condo/migrations/20230412150028-0257_auto_20230412_1000.js

```diff
@@ -0,0 +1,32 @@
+// auto generated by kmigrator
+// KMIGRATOR:0257_auto_20230412_1000:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDQuMSBvbiAyMDIzLTA0LTEyIDEwOjAwCgpmcm9tIGRqYW5nby5kYiBpbXBvcnQgbWlncmF0aW9ucwoKCmNsYXNzIE1pZ3JhdGlvbihtaWdyYXRpb25zLk1pZ3JhdGlvbik6CgogICAgZGVwZW5kZW5jaWVzID0gWwogICAgICAgICgnX2RqYW5nb19zY2hlbWEnLCAnMDI1Nl9yZW1vdmVfYmFua2FjY291bnRfcmVwb3J0dmlzaWJsZV9hbmRfbW9yZScpLAogICAgXQoKICAgIG9wZXJhdGlvbnMgPSBbCiAgICBdCg==
+
+exports.up = async (knex) => {
+    await knex.raw(`
+        BEGIN;
+            --
+            -- [CUSTOM] Set Statement Timeout to some large amount - 25 min (25 * 60 => 1500 sec)
+            --
+            SET statement_timeout = '1500s';  
+            UPDATE "OrganizationEmployeeRole"
+            SET "canManageBankAccountReports" = true
```

Comment:
It's an incorrect syntax of SET command. You should use only one `SET` word for each update operation